extension/data/background/handlers/webrequest.js

```diff
@@ -0,0 +1,123 @@
+/* global messageHandlers */
+'use strict';
+
+/**
+ * Retrieves the user's OAuth tokens from cookies.
+ * @param {number?} [tries=1] Number of tries to get the token (recursive)
```

Comment:
The `[]` denotes optional. Question mark means nothing
```suggestion
 * @param {number} [tries=1] Number of tries to get the token (recursive)
```

Reply:
This was unchanged from before this PR, will update